### DISCUSSION TITLE
Should we add "Feature - Added ability to set placeholder global limit on children only #6847" to the docs?

### DISCUSSION THREAD

**Discussion by TiredFingers** [django-cms/django-cms#7077] [General]
In release 3.9 was added feature - ability to set placeholder global limit on children only #6847
Should we add it to the docs?

**Comment by marksweb** [comment]
Yes 👍

**Comment by NicolaiRidani** [comment]
Well, I guess there's your answer @TiredFingers ;-P

**Comment by crydotsnake** [comment]
Yes 👍🏼

**Comment by gabrielcandrade** [comment]
If it's been tested and already develop the adjust, we can append on this docs version.

I Agree! :)
